cms/jinja2/templates/components/streamfield/quote_block.html

```diff
@@ -1,4 +1,11 @@
-<blockquote>
-    <p>{{ value.quote }}</p>
-    {% if value.attribution %}{{ value.attribution }}{% endif %}
-</blockquote>
+{% from "components/quote/_macro.njk" import onsQuote %}
+{% if value.attribution %}
+    {{-
+    onsQuote({
+    "text": value.quote,
+    "ref": value.attribution
+    })
+    -}}
```

Comment:
Formatting issue.

Reply:
Fixed in e06092cbd6c342e0953e887e5ff8ed1c8764c6cc